uv.gyp

```diff
@@ -104,6 +104,13 @@
             'src/win/winsock.c',
             'src/win/winsock.h',
           ],
+          'conditions': [
+            ['MSVS_VERSION < "2015"', {
```

Comment:
The semi-official way of writing this is `MSVS_VERSION[0:4] < "2015"`, although I think your way should work fine as well.